data/json/recipes/armor/head.json

```diff
@@ -1333,6 +1333,20 @@
     "components": [ [ [ "rag", 8 ] ] ],
     "flags": [ "BLIND_HARD" ]
   },
+  {
+    "result": "welding_mask",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_HEAD",
+    "skill_used": "fabrication",
+    "difficulty": 5,
+    "time": "540 m",
+    "autolearn": true,
+    "using": [ [ "forging_standard", 16 ], [ "steel_standard", 4 ] ],
+    "qualities": [ { "id": "ANVIL", "level": 3 }, { "id": "HAMMER", "level": 3 }, { "id": "CHISEL", "level": 3 } ],
+    "tools": [ [ [ "tongs", -1 ] ], [ [ "crucible", -1 ], [ "crucible_clay", -1 ] ] ],
+    "components": [ [ [ "goggles_welding", 1 ] ] ]
```

Comment:
If you're just sticking some goggles in a frame you've made, I'd prefer the result be a makeshift welding mask, not the same thing as the presumably commercially made welding mask that is lootable.